src/app/camping-vliegenbos/layout.tsx

```diff
@@ -0,0 +1,133 @@
+'use client'
+
+import NextLink from 'next/link'
+import {
+  Grid,
+  Heading,
+  LinkList,
+  Page,
+  PageFooter,
+  PageHeader,
+  Paragraph,
+  SkipLink,
+} from '@amsterdam/design-system-react'
+import { CheckMarkIcon, FacebookIcon, InstagramIcon, MailIcon, PhoneIcon } from '@amsterdam/design-system-react-icons'
+import navigationMenuItems from './navigationMenuItems'
+
+import formatPath from '../../utils/formatPath'
+
+const menuLinks = [
+  {
+    href: formatPath('/camping-vliegenbos/reserveren'),
+    label: 'Reserveren / Reservation',
+  },
+]
+
+export default function CampingVliegenbos({ children }: { children: React.ReactNode }) {
+  return (
+    <Page className="ams-theme">
+      <Grid>
+        <Grid.Cell span="all">
+          <SkipLink href="#main">Direct naar inhoud</SkipLink>
+        </Grid.Cell>
+      </Grid>
+      <PageHeader
+        brandName="Camping Vliegenbos"
+        logoLinkComponent={(props) => <NextLink {...props} href="/camping-vliegenbos" />}
+        logoLinkTitle="Naar de homepage van Camping Vliegenbos"
+        menuItems={menuLinks.map((item, idx) => (
+          <PageHeader.MenuLink href={item.href} key={`${idx}-nav-item`}>
+            {item.label}
+          </PageHeader.MenuLink>
+        ))}
+      >
+        <Grid paddingBottom="large" paddingTop="large">
+          {navigationMenuItems.map((column, idx) => (
+            <Grid.Cell span={3} key={`${idx}-big-menu-column`}>
+              <Heading level={3} className="ams-mb-s">
+                {column.heading}
+              </Heading>
+              <LinkList>
+                {column.items.map((item, itemIdx) => (
+                  <LinkList.Link href={formatPath(item.href)} key={`${idx}-big-menu-column-item-${itemIdx}`}>
+                    {item.label}
+                  </LinkList.Link>
+                ))}
+              </LinkList>
+            </Grid.Cell>
+          ))}
+        </Grid>
+      </PageHeader>
+      <main id="main">{children}</main>
+      <PageFooter>
+        <PageFooter.Spotlight>
+          <Grid gapVertical="2x-large" paddingVertical="x-large">
+            <Grid.Cell span={4}>
+              <Paragraph className="ams-mb-m" color="inverse">
+                Camping Vliegenbos <br />
+                Meeuwenlaan 138 <br />
+                1022 AM Amsterdam
```

Comment:
I think I'd use 3 Paragraphs here, to avoid having to use `br`. [`br`s are not recommended. ](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/br#accessibility)

Reply:
Would you say it applies to this context? The article you link allows the example of an address explicitly:

> The `<br>` HTML element […] is useful for writing a poem or an address, where the division of lines is significant.

It also shows an example of an address, right below the section on accessibility.

I think they mean (far) longer paragraphs than parts of an address when they say

> Creating separate paragraphs of text using <br> is not only bad practice, it is problematic